Source/engine/render/text_render.cpp

```diff
@@ -206,8 +102,8 @@ int GetLineWidth(string_view text, GameFontTables size, int spacing, int *charac
 		if (text[i] == '\n')
 			break;
 
-		uint8_t frame = FontFrame[size][FontIndex[static_cast<uint8_t>(text[i])]];
-		lineWidth += FontKern[size][frame] + spacing;
+		uint8_t frame = text[i] & 0xFF;

```

Comment:
True, this was done early on when I expected to implement Unicode support in the same go.

Reply:
If you think we'll need it when adding Unicode support, then perhaps we can leave it alone. Maybe keep it as a reminder since it's not hurting anything.